tests/backend_test.py

```diff
@@ -63,7 +63,7 @@ def test_quantinuum_sim_h11sc(azure_backend: AzureBackend) -> None:
         counts = r.get_counts()
         assert sum(counts.values()) == 1000
     else:
-        raise ValueError(
+        warnings.warn(  # noqa: B028
```

Comment:
I'm not seeing these warnings in the logs. Not clear if this is because the devices are online now or if they're being suppressed?

Reply:
I think this is a good enough argument to adopt `pytest.skip` instead so it is always clear from the logs what happened